src/components/datagrid/utils/grid_height_width.ts

```diff
@@ -38,16 +38,18 @@ export const useFinalGridDimensions = ({
   // Set the wrapper height on load, whenever the grid wrapper resizes, and whenever rowCount changes
   useEffect(() => {
     const boundingRect = wrapperRef.current!.getBoundingClientRect();
+    const wrapperHeight = wrapperRef.current?.offsetHeight;
+    const wrapperWidth = wrapperRef.current?.offsetWidth;
 
     if (isFullScreen) {
       setFullScreenHeight(boundingRect.height);
```

Comment:
TBH, I think we could probably just make this use `offsetHeight` too and just get rid of `getBoundingClientRect()` altogether - I can't imagine why we'd need the transform-aware a boundingRect in full screen mode
```suggestion
      setFullScreenHeight(wrapperHeight);
```